test/Dialect/SPIRV/Serialization/bin_ops.mlir

```diff
@@ -0,0 +1,61 @@
+// RUN: mlir-translate -serialize-spirv %s | mlir-translate -deserialize-spirv | FileCheck %s
+// CHECK:           func {{@.*}}([[ARG1:%.*]]: f32, [[ARG2:%.*]]: f32) {
```

Comment:
No need to check the `func`s; just need to check the arithmetic ops. Checks in the test should focus on what we are testing. Checking additional stuff will make the test difficult to maintain in the future (say if we change how `func` are printed then we need to update everywhere, even in this test case, which is not related to testing `func` anyway.)

Besides, we typically add the `// CHECK` to be directly above the line to be checked for easy contrast. :)

Reply:
Thanks for the explanation, fixed.